Gems/PhysX/Code/Source/Articulation.cpp

```diff
@@ -23,9 +23,9 @@ namespace PhysX
         if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serializeContext->Class<ArticulationLinkData>()
-                ->Version(1)
+                ->Version(2)
                 ->Field("LinkConfiguration", &ArticulationLinkData::m_articulationLinkConfiguration)
-                ->Field("ShapeColliderPair", &ArticulationLinkData::m_shapeColliderConfiguration)
+                ->Field("ShapeColliderPairList", &ArticulationLinkData::m_shapeColliderConfigurationList)
```

Comment:
Will changing this break any existing data? Does anything need to be done about converting older versions?

Reply:
I think there is no need to convert the version here. This structure is created from the articulation hierarchy during creating spawnable from prefab. It will probably break binary spawnable (it will cause the 
/need to regenerate product assets), but it will not break any prefab. It is not a field of any Editor Component, it is generated from entity data with the utility function EntityDataToArticulationLinkData.
Please also take a look into prefab that utilizes articulations:
https://github.com/o3de/o3de-extras/blob/development/Templates/Ros2RoboticManipulationTemplate/Template/Prefabs/PandaRobot.prefab, it does not use the abovementioned structure.